airbyte-ci/connectors/connectors_qa/connectors_qa/consts.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from base_images.python.bases import AirbytePythonConnectorBaseImage  # type: ignore
+
+CONNECTORS_QA_DOC_TEMPLATE_NAME = "connectors_qa.md.j2"
+DOCKER_HUB_PASSWORD_ENV_VAR_NAME = "DOCKER_HUB_PASSWORD"
+DOCKER_HUB_USERNAME_ENV_VAR_NAME = "DOCKER_HUB_USERNAME"
+DOCKER_INDEX = "docker.io"
+DOCKERFILE_NAME = "Dockerfile"
+DOCUMENTATION_STANDARDS_URL = "https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw"
+GRADLE_FILE_NAME = "build.gradle"
```

Comment:
How is Gradle used internally, and for what?

Reply:
@natikgadzhi these checks run also on our java connectors which are using gradle. 
This file is used in this program to determine if a connector is a java one.